pydis_site/apps/api/viewsets/bot/nomination.py

```diff
@@ -213,6 +268,11 @@ def partial_update(self, request: HttpRequest, *args, **kwargs) -> Response:
                     {'end_reason': ['This field is required when ending a nomination.']}
                 )
 
+            if 'reviewed' in request.data:
+                raise ValidationError(
+                    {'reviewed': ['This field cannot be set same time than ending nomination.']}
```

Comment:
I don't understand what this means, but I think this is an improvement. (It may be the opposite).
```suggestion
                    {'reviewed': ['This field cannot be set while you are ending a nomination.']}
```

Reply:
Make sure to change the test as well.